alembic/versions/38c9c18d342e_add_new_mdapi_rule.py

```diff
@@ -0,0 +1,57 @@
+"""Add new mdapi rule.
+
+Revision ID: 38c9c18d342e
+Revises: 362efe8fd524
+Create Date: 2015-11-13 13:49:25.020563
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '38c9c18d342e'
+down_revision = '362efe8fd524'
+
+from alembic import op
+import sqlalchemy as sa
+
+path = 'fmn.rules:mdapi_repo_update'
+target = "Events on packages that I own"
+
+
+def upgrade():
+    engine = op.get_bind().engine
+    session = sa.orm.scoped_session(sa.orm.sessionmaker(bind=engine))
+
+    valid_paths = fmn.lib.load_rules(root='fmn.rules')
+
+    filters = session.query(fmn.lib.models.Filter).filter_by(name=target).all()
+    print "Found %i filters" % len(filters)
+
+    modified = 0
+    for filt in filters:
+        if len(filt.rules) < 15:
+            # Someone has changed this filter dramatically, so let's not mess
+            # with it.
+            print "Avoiding %r on %r.  Only %i rules present." % (
+                filt, filt.preference, len(filt.rules))
+            continue
+
+        modified += 1
+        filt.add_rule(session, valid_paths, path, negated=True)
```

Comment:
@pypingou see here.  We're adding this rule with `negated=True`.. so we're adding the rule to *exclude* this message from people's filters.  Make sense?